packages/services/f-test-data-creator/src/test-data-generator.js

```diff
@@ -1,14 +1,13 @@
-/* eslint-disable class-methods-use-this */
 const crypto = require('crypto');
 
 module.exports = class TestDataGenerator {
-    get firstName () { return 'John'; }
+    static get firstName () { return 'John'; }
 
-    get lastName () { return 'Testington'; }
+    static get lastName () { return 'Testington'; }
 
-    get emailAddress () { return this._emailAddress; }
+    static get emailAddress () { return this._emailAddress; }
```

Comment:
Does it matter that the getter is static and the setter isn't?

Reply:
Also does referencing `this`cause any issues?

Reply:
I think this one can be removed (I did a big find and replace, so any getter's with references to this. shouldn't need static added to them).

Reply:
Just as a side note – I had to get rid of all the static references as it killed all the tests 🥴